### PR TITLE
PLANET-6080 Move fonts out of CSS file

### PR DESCRIPTION
**Pull request by Inwerpsel**
Ref: https://jira.greenpeace.org/browse/PLANET-6080

---
* Moved to [HTML head in the theme](https://github.com/greenpeace/planet4-master-theme/pull/1364/files#diff-a40ff5ae673d9e734e86aef984e9588608dee8d8e31ee3da3a1a5b845101ea3cR3-R5), so that Cloudflare can optimize them.
* Preserve in a separate file so the editor can still load them.

### CODE REVIEW

File: assets/src/styles/editorStyle.scss [hunk 1]
```diff
@@ -26,6 +26,7 @@
 @import "components/Preview";
 @import "components/CharacterCounter";
 @import "campaigns/base/fonts";
+@import "campaigns/base/google_fonts";
```
Comment (lithrel) [inline assets/src/styles/editorStyle.scss:29]: Should we add the same to [`assets/src/styles/campaigns.scss`](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/f918ee7627cfe4329b202d5ec33d1c122ccfe887/assets/src/styles/campaigns.scss#L11) ?

**Review by lithrel — APPROVED** [review]
:+1: 
While checking this I realized the title of the Antarctic campaign in the editor doesn't have the Sanctuary font applied, and I clearly remember seeing it before, I wonder when we lost this or if it was on purpose, or if I imagined things ?

**Comment by Inwerpsel** [comment]
> While checking this I realized the title of the Antarctic campaign in the editor doesn't have the Sanctuary font applied (doesn't come from this PR), and I clearly remember seeing it before, I wonder when we lost this or if it was on purpose, or if I imagined things ?

Probably [this part that relies on `DOMContentLoaded`](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/5df0e0378749f1d5fd0b03f22ca545152ba953dc/assets/src/connectCssVariables.js#L46-L52) got broken.

**Comment by Inwerpsel** [comment]
> > While checking this I realized the title of the Antarctic campaign in the editor doesn't have the Sanctuary font applied (doesn't come from this PR), and I clearly remember seeing it before, I wonder when we lost this or if it was on purpose, or if I imagined things ?
> 
> Probably [this part that relies on `DOMContentLoaded`](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/5df0e0378749f1d5fd0b03f22ca545152ba953dc/assets/src/connectCssVariables.js#L46-L52) got broken with a WP version update that caused the event to fire earlier. There's definitely a better way to write it.

I made it work again locally, but now it works too well :grin: 
Screenshot from 2021-04-21 17-12-50

**Pull request merged**